doc/changes/latest.inc

```diff
@@ -300,8 +300,6 @@ API
 
 - ``mne.io.pick.get_channel_types`` is now called ``mne.io.pick.get_channel_type_constants`` to better reflect its return values, by `Daniel McCloy`_.
 
-- Methods :meth:`mne.io.Raw.get_channel_types`, :meth:`mne.Epochs.get_channel_types`, and :meth:`mne.Evoked.get_channel_types` now accept parameters ``picks``, ``unique``, and ``only_data_chs``, by `Daniel McCloy`_.
```

Comment:
these methods were added in 0.20.dev0, so no need to call out the parameter changes.